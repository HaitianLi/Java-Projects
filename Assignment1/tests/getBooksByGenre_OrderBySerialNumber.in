add collection books.csv
add collection books2.csv
genre Genre testcase
exit