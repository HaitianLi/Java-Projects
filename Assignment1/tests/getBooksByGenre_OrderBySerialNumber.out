user: 17 books successfully added.

user: 19 books successfully added.

user: first show with 211120 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
third show with 211203 (J.K. Rowling)
final with 211204 (Bob Kane)

user: Ending Library process.
